src/mongocxx/test/uri.cpp

```diff
@@ -71,6 +71,11 @@ TEST_CASE("URI", "[uri]") {
             mongocxx::uri{invalid};
         } catch (const mongocxx::logic_error& e) {
             REQUIRE(e.code() == mongocxx::error_code::k_invalid_uri);
+
+            std::string invalid_schema =
+                "Invalid URI Schema, expecting 'mongodb://' or 'mongodb+srv://': ";
+
+            REQUIRE(e.what() == invalid_schema + e.code().message());
```

Comment:
Not sure if this test is necessary but I added it in the moment to make sure we were successfully getting the correct error message.